help_to_heat/frontdoor/interface.py

```diff
@@ -452,11 +454,11 @@ def parse_dpa_to_address(self, dpa):
 
 class EPC(Entity):
     @with_schema(load=GetEPCSchema, dump=EPCSchema)
-    def get_epc(self, uprn, country):
+    def get_epc_scotland(self, uprn, country):
         try:
-            if country == "England" or country == "Wales":
-                epc = portal.models.EpcRating.objects.get(uprn=uprn)
-            elif country == "Scotland":
+            # if country == "England" or country == "Wales":
+            #     epc = portal.models.EpcRating.objects.get(uprn=uprn)
```

Comment:
This is only commented out for now as I wanted to leave it in as a reference to the `EpcRating` object which I'll need to remove everywhere